docs/installation/server-config/security.rst

```diff
@@ -1,7 +1,7 @@
 .. _ksql-security:
 
 Configuring Security for KSQL
-=============
+=============================
```

Comment:
Thanks for paying attention to this! (to ensure Sphinx isn't spewing errors/warnings at us)